criu/config.c

```diff
@@ -92,9 +92,10 @@ int parse_statement(int i, char *line, char **configuration)
 			}
 		}
 
-		offset += strlen(configuration[i]);
+		if (configuration[i])
```

Comment:
can you explain when configuration[i] can be null here?

Reply:
`} else if ((configuration[i] = strtok_r(line + offset, "\"", &quotedptr))) {` could assign NULL to configuration[i]

Reply:
We need to write a comment what it means here. If we expect to see null here, we need to describe a case when this can happen. If we don't expect to see NULL here, we need to report an error and log it.

Reply:
I had a closer look at this and this is a false positive. In theory the report is correct, but strtok is searching for `"` which is checked before to be there, so strtok will always find something and never return NULL. These checks are not necessary. I dropped the patch.